app/jobs/delete_old_searches_job.rb

```diff
@@ -0,0 +1,9 @@
+##
+# Deletes from the database searches older than 1 week
+class DeleteOldSearchesJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(*args)
+    Search.destroy_all(['updated_at < ?', Time.now - 1.week])
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.